src/backend/joanie/tests/core/test_api_contract.py

```diff
@@ -1474,7 +1474,7 @@ def test_api_contract_generate_zip_archive_authenticated_post_method_allowed(sel
                 payment_schedule=[
                     {
                         "amount": "200.00",
-                        "due_date": "2024-01-17T00:00:00+00:00",
+                        "due_date": "2024-01-17",
```

Comment:
Ditto

Reply:
the factory now takes care of it :sunglasses: